### PR TITLE
CDAP-537: Allow Users of common-cli to capture UserInterruptException to provide a handler method to handle Ctrl+C

### PR DESCRIPTION
**Pull request by artemmakogon**
@gokulavasan Please review

### CODE REVIEW

**Comment by gokulavasan** [comment]
@artemmakogon @jwang47 Do you know if there is a cleaner way to test this. If there is no other option other than to use Reflection, LGTM :+1:

File: common-cli/src/test/java/co/cask/common/cli/InterruptHandlersTest.java [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+import java.lang.reflect.Method;
+import java.util.Collections;
+
+public class InterruptHandlersTest {
+  private boolean isUserInterruptHandled;
+
+  @Test
+  public void test() throws Exception {
+    CLI cli = new CLI(Collections.emptyList(), Collections.emptyMap());
+    char ctrlBreak = (char) 3;
```
Comment (jwang47) [inline common-cli/src/test/java/co/cask/common/cli/InterruptHandlersTest.java]: Is a constant for this?

**Comment by jwang47** [comment]
One question. Otherwise LGTM. Please make sure the build passes.

**Comment by artemmakogon** [comment]
@jwang47 I've updated PR. Thank you

**Pull request merged**